.eslintrc

```diff
@@ -1,11 +1,16 @@
 {
-  "extends": ["airbnb", "plugin:jest/recommended"],
-  "plugins": ["jest"],
+  "extends": ["airbnb", "prettier", "plugin:jest/recommended"],
+  "plugins": ["prettier", "jest"],
   "env": {
     "browser": true,
     "jest/globals": true
   },
   "rules": {
+    "max-len": ["error", 80, {
```

Comment:
tells eslint that > 80 line length is an error, not just a warning. Except import statements, which is prettier's opinion.